Documentation/operations/performance/tuning.rst

```diff
@@ -699,6 +699,29 @@ distributions offer kernel images with this option set or you can re-compile
 the Linux kernel. ``CONFIG_PREEMPT_NONE=y`` is the recommended setting for
 server workloads.
 
+Scaling
+=======
+
+By default, the `cilium daemonset <https://github.com/cilium/cilium/blob/main/install/kubernetes/cilium/templates/cilium-agent/daemonset.yaml>`_
```

Comment:
I don't think a link to the template is really useful here, but if you think it is, I would rather make it a permalink.

Reply:
Sounds good, I'll remove it.